src/Microsoft.Health.Fhir.Core/Extensions/LoggerExtensions.cs

```diff
@@ -42,10 +43,21 @@ public static void LogVerbose(this ILogger logger, IParameterStore store, Cancel
 
 #pragma warning disable CA1849 // Runs synchronously to allow for the logger to be used in a synchronous context.
             if (logParameterTask.Result.CharValue == "Verbose")
-#pragma warning restore CA1068
+#pragma warning restore CA1849
             {
                 logger.LogInformation(message, args);
             }
         }
+
+        public static (Stopwatch sw, string label) StartStopwatch(this ILogger logger, string label)
```

Comment:
Health shared has ActionTimer, that is kind of like this but uses an IDisposable to automatically start/stop the timer - https://github.com/microsoft/healthcare-shared-components/blob/d039a4e72fa3988f490f9349f674b17808055038/src/Microsoft.Health.Core/ActionTimerExtensions.cs